webshare/share-cancel-manual.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Manual Test: Share cancelled by user</title>
```

Comment:
It's OK to omit "WebShare Manual Test" from all titles, this is clear from context.